### PR TITLE
Updates to Wordle text and yellow highlight

### PR DESCRIPTION
**Pull request by 3lkn**
Updated the Wordle text to slightly more friendly interaction.
Changed yellow highlight to not over-highlight letters (don't count dupes of "correct letter in correct spot" as "correct letter in incorrect spot").

### CODE REVIEW

File: examples/wordle/core/src/lib.rs [hunk 1]
```diff
@@ -49,6 +50,6 @@ impl WordFeedback {
             }
             print!("{:}", guess_word.chars().nth(i).unwrap());
         }
-        println!("\x1b[0m");
+        println!("\x1b[0m\n");
```
Comment (flaub) [inline examples/wordle/core/src/lib.rs:53]: This is weird, because `println!` includes a `\n`

Comment (flaub) [reply examples/wordle/core/src/lib.rs:53]: If you want a newline, I think it's clearer to say:
```
println!("\x1b[0m");
println!("");
```

**Review by flaub — APPROVED** [review]
Thanks for adding a test for the false-positive case.

File: examples/wordle/src/main.rs [hunk 2]
```diff
@@ -105,22 +114,24 @@ fn play_rounds(server: Server, player: Player, rounds: usize) -> bool {
...
+    println!("Welcome to fair Wordle! Enter a five-letter word.");
 
     let server = Server::new(wordlist::pick_word());
     let player = Player {
         hash: server.get_secret_word_hash(),
     };
 
     if play_rounds(server, player, 6) {
-        println!("You won!");
+        println!("You won!\n");
```
Comment (flaub) [inline examples/wordle/src/main.rs:125]: I think we should do the same for all these cases

**Review by tzerrell — APPROVED** [review]
great!

**Comment by flaub** [comment]
Bump, would be good to get this landed sooner than later

**Pull request merged**